### PR TITLE
Modified the group members' page ...

### PR DESCRIPTION
**Pull request by rgilman**
... so groups under 22 display self-intros. Larger groups display avatars as in the main member directory page

### CODE REVIEW

**Comment by rgilman** [comment]
Compare https://com.context.org/groups/bfcom-team/members/all-members/ with https://bbp.context.org/groups/bfcom-team/members/all-members/

File: buddypress/groups/single/members-loop.php [hunk 1]
```diff
@@ -30,16 +31,17 @@
...
 
-	<ul id="members-list" class="<?php bp_nouveau_loop_classes(); ?>">
+	<script> 
+		jQuery(document).foundation();
+	</script>
 
-		<?php
+	<?php
+	$count = groups_get_current_group()->total_member_count;
+	If ($count < 22) : ?>
```
Comment (iangilman) [inline buddypress/groups/single/members-loop.php:40]: I guess PHP isn't case-sensitive? Normally I'd say that "if" should be lowercase.

**Review by iangilman — COMMENTED** [review]
That's a neat idea, to have the extra info if you got the space for it :-)

File: buddypress/groups/single/members-loop.php (hunk 1 above)
Comment (rgilman) [reply buddypress/groups/single/members-loop.php:40]: Good catch. Apparently not. :-) 

More specifically, it looks like variables and constants are case-sensitive while functions and methods aren't.

Nevertheless, good form to keep all the functions as lower-case. I've fixed this in my local version and will include the fix in the next push. I did a global search in the theme and found only this one instance of `If ( `. I'll try to be conscious of this in the future.

Comment (iangilman) [reply buddypress/groups/single/members-loop.php:40]: Sounds good :-)

**Pull request merged**